Makefile.pre.in

```diff
@@ -2388,6 +2388,7 @@ TESTSUBDIRS=	idlelib/idle_test \
 		test/support \
 		test/support/_hypothesis_stubs \
 		test/support/interpreters \
+		test/ast_repr_data \
```

Comment:
I would prefer to put test_ast data in Lib/test/test_ast/, no?

Reply:
Yeah now that it's a folder it make sense to keep it there